tron/core/actionrun.py

```diff
@@ -150,6 +150,10 @@ class ActionRun(object):
     STATE_FAILED['skip'] = STATE_SKIPPED
     STATE_CANCELLED['skip'] = STATE_SKIPPED
 
+    STATE_UNKNOWN['running'] = STATE_RUNNING
+    STATE_UNKNOWN['failed'] = STATE_FAILED
```

Comment:
There's some magic in `__getattr__` that makes all of the transition names correspond to a method on an action run.  This should be 'fail' since we already have a method called that, which handles that transition. (It's also done below in L157-160, but I don't mind making it explicit here.)

Reply:
oh, great, thanks.